dietpi/dietpi-software

```diff
@@ -6649,6 +6659,34 @@ _EOF_
 
 		fi
 
+		#HTPC Manager
+		INSTALLING_INDEX=152
+		if (( ${aSOFTWARE_INSTALL_STATE[$INSTALLING_INDEX]} == 1 )); then
+
+			Banner_Installing
+
+			INSTALL_URL_ADDRESS='https://github.com/Hellowlol/HTPC-Manager.git'
+
+			/DietPi/dietpi/func/check_connection "$INSTALL_URL_ADDRESS"
+			#Install
+			if (( $? == 0 )); then
+
+				#Install Python and PIP
+				AGI python python-pip python-imaging
+
+				cd "$HOME"
```

Comment:
Not required as this is called before the start of all software installs, however, no harm.